commands.go

```diff
@@ -59,6 +59,69 @@ func (h hostListItemByName) Less(i, j int) bool {
 	return strings.ToLower(h[i].Name) < strings.ToLower(h[j].Name)
 }
 
+func beforeFunc(c *cli.Context) error {
```

Comment:
Should we name this something like `createBefore` or something to show that it's tied to the `Create`?  `beforeFunc` seems a little generic.